phpstan-baseline.neon

```diff
@@ -3855,6 +3855,21 @@ parameters:
 			count: 1
 			path: tests/Backend/Workspaces/Backend/SnowflakeWorkspaceBackend.php
 
+		-
+			message: "#^Cannot access offset 'schema' on mixed\\.$#"
+			count: 1
+			path: tests/Backend/Workspaces/Backend/SnowflakeWorkspaceBackendDBAL.php
+
+		-
+			message: "#^Parameter \\#1 \\$connection of method Keboola\\\\Test\\\\Backend\\\\Workspaces\\\\Backend\\\\SnowflakeWorkspaceBackendDBAL\\:\\:getDbConnectionSnowflakeDBAL\\(\\) expects array, mixed given\\.$#"
+			count: 1
+			path: tests/Backend/Workspaces/Backend/SnowflakeWorkspaceBackendDBAL.php
+
+		-
+			message: "#^Property Keboola\\\\Test\\\\Backend\\\\Workspaces\\\\Backend\\\\SnowflakeWorkspaceBackendDBAL\\:\\:\\$schema \\(string\\) does not accept mixed\\.$#"
+			count: 1
+			path: tests/Backend/Workspaces/Backend/SnowflakeWorkspaceBackendDBAL.php
+
```

Comment:
tady to je na prd, ale když sem tam začal doplňovat to schéma tak bych to musel udělat všude jinak bych do baseline přidal asi dalších 1000 záznamů :(